fh_immuta_utils/data_source.py

```diff
@@ -53,11 +53,12 @@ def make_immuta_datasource_name(
     Returns a table name that's guaranteed to be unique and within the Immuta data source name max char limit (255)
     """
     table_name = ""
-    if user_prefix:
-        table_name = f"{user_prefix}_"
-    table_name += f"{PREFIX_MAP[handler_type]}_{schema}_{table}"
-    if table_name is None:
-        return None
+    table_name += f"{user_prefix}_" if user_prefix else ""
+    table_name += f"{PREFIX_MAP[handler_type]}_" if handler_type else ""
```

Comment:
Minor, do we want to make typing of `handler_type` and `schema` as `Optional[str]` like `user_prefix`?

Reply:
thanks for noticing this. i forgot to go back and remove these conditional statements so that we can continue to preserve unique data sources names. updated